src/coreclr/tools/cdac-build-tool/Program.cs

```diff
@@ -0,0 +1,42 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Buffers.Binary;
+using System.Collections.Generic;
+using System.CommandLine;
+using System.CommandLine.Help;
+using System.CommandLine.Parsing;
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Reflection;
+using System.Reflection.Metadata;
+using System.Reflection.Metadata.Ecma335;
+using System.Reflection.PortableExecutable;
+using System.Runtime.Serialization.Json;
+using System.Text;
+using System.Text.Json;
+using System.Text.Encodings.Web;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
```

Comment:
nit: delete unused usings on C# sources to reduce the noise.

Reply:
@am11 thanks.  this change is actually part of #100650 (PTAL if you're interested) - and I already changed that one to remove the unused usings.  I'll rebase this current PR so it doesn't include changes from that one.  I thought I might end up sharing code, but it didn't seem useful yet.